src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">AnySoftKeyboard - change_me Language Pack</string>
-    <string name="keyboard_name">change_me keyboard</string>
-    <string name="dictionary_name">change_me</string>
+    <string name="app_name">AnySoftKeyboard - Indonesian Language Pack</string>
+    <string name="keyboard_name">Indonesian keyboard</string>
```

Comment:
remove the "keyboard" bit, because this string will show up on the select button